spec/fuzzy-finder-spec.js

```diff
@@ -93,11 +93,19 @@ describe('FuzzyFinder', () => {
     }
   }
 
-  for (const useRipGrep of [false, true]) {
-    describe(`file-finder behavior (ripgrep=${useRipGrep})`, () => {
+  const testPermutations = [
+    [false, 'standard'],
+    [true, 'standard'],
+    [false, 'alternate']
+    [false, 'fast']
```

Comment:
I don't think it's worth to test on every single permutation of `useRipGrep:scoringSystem`, since these two params are orthogonal, so I've just selected a few permutations that ensure that both variables get fully tested.